CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ NOTES:
 
 * provider: The underlying Terraform codebase dependency for the provider SDK and acceptance testing framework has been migrated from `github.com/hashicorp/terraform` to `github.com/hashicorp/terraform-plugin-sdk`. They are functionality equivalent and this should only impact codebase development to switch imports. For more information see the [Terraform Plugin SDK page in the Extending Terraform documentation](https://www.terraform.io/docs/extend/plugin-sdk.html). [GH-10367]
 
+FEATURES:
```

Comment:
Please note that the maintainers are responsible for updating the CHANGELOG.md in this project, to prevent merge conflicts. 😄

Reply:
Removed, thanks!